### PR TITLE
Updated AddressValidation# url with hostname

### PR DESCRIPTION
**Pull request by RachalCassity**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

Added another log and changed the AddressValidation URL to AddressValidation hostname.

## Related issue(s)
- department-of-veterans-affairs/va.gov-team/issues/98561

## Testing done

- [x] *New code is covered by unit tests*
- *Describe what the old behavior was prior to the change*
- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing*
- *If this work is behind a flipper:*
  - *Tests need to be written for both the flipper on and flipper off scenarios. [Docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Backendexample).*
  - *What is the testing plan for rolling out the feature?*

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: lib/va_profile/v3/address_validation/configuration.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ module V3
     module AddressValidation
       class Configuration < VAProfile::Configuration
         def base_path
-          "#{VAProfile::Configuration::SETTINGS.address_validation.url}/services/address-validation/v3/"
+          "https://#{VAProfile::Configuration::SETTINGS.address_validation.hostname}/services/address-validation/v3/"
```
[the remark this answers is not in the document]
Comment (RachalCassity) [reply lib/va_profile/v3/address_validation/configuration.rb:10]: It is not the same in other environments

[the remark this answers is not in the document]
Comment (RachalCassity) [reply lib/va_profile/v3/address_validation/configuration.rb:10]: In other environments, the urls do not point to sandbox

[the remark this answers is not in the document]
Comment (rmtolmach) [reply lib/va_profile/v3/address_validation/configuration.rb:10]: oh, I see. `address_validation.url` isn't a value in the manifest repo vets-api envs. Only in settings.yml in vets-api.

[the remark this answers is not in the document]
Comment (RachalCassity) [reply lib/va_profile/v3/address_validation/configuration.rb:10]: Yes, oops lol

**Review by ericboehs — APPROVED** [review]

**Pull request merged**